homeassistant/components/homee/entity.py

```diff
@@ -0,0 +1,126 @@
+"""Base Entities for Homee integration."""
+
+from pyHomee.const import AttributeType, NodeProfile
+from pyHomee.model import HomeeAttribute, HomeeNode
+
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import Entity
+
+from . import HomeeConfigEntry
+from .const import DOMAIN
+from .helpers import get_name_for_enum
+
+
+class HomeeNodeEntity(Entity):
+    """Representation of a Node in Homee."""
+
+    _attr_has_entity_name = True
+
+    def __init__(self, node: HomeeNode, entry: HomeeConfigEntry) -> None:
+        """Initialize the wrapper using a HomeeNode and target entity."""
+        self._node = node
+        self._clear_node_listener = None
+        self._attr_unique_id = node.id
+        self._entry = entry
```

Comment:
Why do we store the entry?

Reply:
because we need `homee` from `runtime_data` in functions `async_update` & `async_set_value_by_id`